app/assets/stylesheets/views/_detailed-guide.scss

```diff
@@ -19,4 +19,14 @@
       margin: 0 0 $gutter-half $gutter-half;
     }
   }
+
+  // To allow .back-to-content to settle in at the bottom after scrolling past.
+  .sidebar-with-body {
+    position: relative;
+  }
+
+  // To compensate for the -15px from @include sidebar-with-body.
+  .back-to-content {
+    margin-left: 15px;
```

Comment:
Should that be based on `$gutter`?
I can't see anything referring to `-15px` in `_sidebar-with-body.scss`?

:wave: